lib/measured/measurable.rb

```diff
@@ -1,3 +1,5 @@
+require 'bigdecimal/util'
```

Comment:
My gut is to not do this. One of my goals with this gem is to not modify core types like so many of the others do. This is less bad because it's a stdlib thing, but still means `Float` changes because of this gem.

You can just modify the case statement below to do the `Float::DIG+1` thing.